database/insertData.js

```diff
@@ -97,6 +98,7 @@ const queries = [
                         label: "What category of organisation do you wish to join"
                         options: ["Manufacturer", "Distributor", "Importer"]
                       }
+                      isRequired: false
```

Comment:
Did you mean to make this one `isRequired: false`? Wouldn't it be Q7 (Select Manufacturer) rather than this one, since Q7 is dependent on this (and you've made the other two conditional questions `isRequired: false`

Reply:
I think that's what I wanted. Then I'll be checking the following (for the progress bar):
- Q6: `{ isRequired: false + isVisible: True }`  => So this can be empty and the progress can continue.
- Q7: `{ isRequired: true + isVisible: false }` => If the Q6 is empty this one isn't visible - so the process can continue.
- Q7: `{ isRequired: true + isVisible: true }` => If the Q6 has an answer now this question is expected to be filled.

Reply:
Okay, just not sure why Q7 is required and Q8/9 are not, when all three of them are dependent on the selection in Q6.

Reply:
Just for testing purposes. I wanted to test one question that `isRequired` but can be visible or not. Instead of adding the same to all the 3 I have made only one example to check.

Reply:
Ah, okay. Sweet.